packages/vue-searchbox/src/components/SearchBase.jsx

```diff
@@ -14,11 +14,15 @@ const SearchBase = {
 		transformResponse: VueTypes.func
 	},
 	provide() {
+		const headers = {
+			'x-search-client': 'Searchbox Vue',
+			...this.$props.headers
```

Comment:
```suggestion
		const headers = {
		        ...this.$props.headers,
			'x-search-client': 'Searchbox Vue'
```
@mohdashraf010897 We shouldn't allow users to override the `x-search-client` header.